controllers/dspipeline_controller.go

```diff
@@ -165,8 +165,6 @@ func (r *DSPAReconciler) isDeploymentAvailable(ctx context.Context, dspa *dspav1
 func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.WithValues("namespace", req.Namespace)
 
-	log.V(1).Info("DataSciencePipelinesApplication Reconciler called.")
```

Comment:
you don't need to move this and re-assign the logger, you have access to the name of the custom resource from `req.name`, e.g: 

```go

	log := r.Log.WithValues("namespace", req.Namespace).WithValues("dspa_name", req.Name)
	log.V(1).Info("DataSciencePipelinesApplication Reconciler called.")
	params := &DSPAParams{}

	dspa := &dspav1alpha1.DataSciencePipelinesApplication{}
	err := r.Get(ctx, req.NamespacedName, dspa)
	if err != nil && apierrs.IsNotFound(err) {
		log.Info("Stop DSPAParams reconciliation")
		return ctrl.Result{}, nil
	} else if err != nil {
		log.Error(err, "Unable to fetch the DSPAParams")
		return ctrl.Result{}, err
	}
	// FixMe: Hack for stubbing gvk during tests as these are not populated by test suite
```